src/Page/Food/Builder.elm

```diff
@@ -1046,28 +1044,38 @@ transportToTransformationView selectedImpact recipe results =
         ]
 
 
-transportToPackagingView : Recipe -> Html Msg
-transportToPackagingView recipe =
+transportToPackagingView : Recipe -> Recipe.Results -> Html Msg
+transportToPackagingView recipe results =
```

Comment:
I believe the `Recipe` is "kinda" also in the `Results`, right? Maybe we don't need to pass both the recipe and the results here? Not sure about the data architecture...

Reply:
Hmm no, here with need `Recipe` to access the transform, `results.recipe` is just resulting impacts for just the recipe step, we don't know about the transform. So I'm gonna keep them both.